model/task/test_result.go

```diff
@@ -345,3 +365,116 @@ func sortTestResults(results []testresult.TestResult, opts *FilterOptions, baseS
 		return false
 	})
 }
+
+func (o TestResultOutput) downloadParquet(ctx context.Context, credentials evergreen.S3Credentials, t *testresult.DbTaskTestResults) ([]testresult.TestResult, error) {
+	prestoBucket, err := o.GetPrestoBucket(ctx, credentials)
+	if err != nil {
+		return nil, err
+	}
+
+	r, err := prestoBucket.Get(ctx, t.PrestoPartitionKey())
+	if err != nil {
+		return nil, errors.Wrap(err, "getting Parquet test results")
+	}
+	defer func() {
+		err = r.Close()
+		grip.WarningWhen(err != nil, message.WrapError(err, message.Fields{
```

Comment:
nit: there's no need to check if `err != nil` because grip logs suppress logging when `message.WrapError` wraps a nil error.